dotCMS/src/main/java/com/dotcms/publishing/output/DirectoryBundleOutput.java

```diff
@@ -0,0 +1,111 @@
+package com.dotcms.publishing.output;
+
+import com.dotcms.publishing.BundlerUtil;
+import com.dotcms.publishing.PublisherConfig;
+import com.google.common.annotations.VisibleForTesting;
+import com.liferay.util.FileUtil;
+import org.jetbrains.annotations.NotNull;
+
+import java.io.File;
+import java.io.FileFilter;
+import java.io.IOException;
+import java.io.OutputStream;
+import java.nio.file.Files;
+import java.util.Collection;
+
+public class DirectoryBundleOutput extends BundleOutput {
+    private String lastFilePath;
+    private File lastFile;
+    private File directoryRootPath;
+
+    public DirectoryBundleOutput(final PublisherConfig publisherConfig) {
+        super(publisherConfig);
+        directoryRootPath = BundlerUtil.getBundleRoot( publisherConfig );
+    }
+
+    @VisibleForTesting
+    public DirectoryBundleOutput(final PublisherConfig publisherConfig, final File directoryPath) {
+        super(publisherConfig);
+        directoryRootPath = directoryPath;
+    }
+
+    @Override
+    public boolean useHardLink() {
+        return true;
```

Comment:
shouldn't be this configurable?

Reply:
really it is 

https://github.com/dotCMS/core/blob/e491040a5cc382fa8a25457ba027679b4afa86b7/dotCMS/src/main/java/com/dotcms/publishing/output/BundleOutput.java#L52